tests/test_schema.py

```diff
@@ -70,8 +68,8 @@ def test_unauthorized_database():
 
 
 def test_drop_database():
-    schema = dj.schema(PREFIX + '_drop_test', locals(), connection=dj.conn(**CONN_INFO))
+    schema = dj.schema(PREFIX + '_drop_test', locals(), connection=dj.conn(**CONN_INFO, reset=True))
```

Comment:
This statement leads to an error in 3.4 or lower due to placement of a keyword argument after dictionary expansion. Replace with `dj.conn(reset=True, **CONN_INFO)`